src/server/editorServices.ts

```diff
@@ -3325,7 +3340,7 @@ namespace ts.server {
         }
 
         private telemetryOnOpenFile(scriptInfo: ScriptInfo): void {
-            if (this.syntaxOnly || !this.eventHandler || !scriptInfo.isJavaScript() || !addToSeen(this.allJsFilesForOpenFileTelemetry, scriptInfo.path)) {
+            if (this.serverMode !== LanguageServiceMode.Semantic || !this.eventHandler || !scriptInfo.isJavaScript() || !addToSeen(this.allJsFilesForOpenFileTelemetry, scriptInfo.path)) {

```

Comment:
Probably an issue for a future PR, but I think the assumption here was that syntax server telemetry would be uninteresting.  I'm not sure the same is true for the partial semantic server.